### PR TITLE
✨ Add quick helm deploy command

### PR DESCRIPTION
**Pull request by wang-chunxing**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -6,8 +6,13 @@ VERSION          ?= 0.0.0
 SERVICE_PORT     ?= 3000
 DOCKER_REGISTRY  ?= docker.io/tosone
 DOCKER_PLATFORMS ?= linux/amd64,linux/arm64
-
+APPNAME          ?= sigma
+NAMESPACE        ?= default
```
Comment (tosone) [inline Makefile:10]: namespace use `sigma`, and `helm install --create-namespace` will create this ns automatically.

File: Makefile [hunk 2]
```diff
@@ -121,3 +126,19 @@ help: ## Show this help.
 		if (/^[a-zA-Z_-]+:.*?##.*$$/) {printf "    ${YELLOW}%-30s${GREEN}%s${RESET}\n", $$1, $$2} \
 		else if (/^## .*$$/) {printf "  ${CYAN}%s${RESET}\n", substr($$1,4)} \
 		}' $(MAKEFILE_LIST)
+
+build_push: build
+    ifeq ($(DOCKER_REGISTRY),)
+        $(error DOCKER_REGISTRY is required. Please provide a value.)
+    endif
+	@docker buildx build -f build/Dockerfile --progress plain --output type=image,name=$(DOCKER_REGISTRY)/$(BINARY_NAME):latest,push=true .
```
Comment (tosone) [inline Makefile:134]: add `--platform $(DOCKER_PLATFORMS)` params.

File: Makefile [hunk 3]
```diff
@@ -121,3 +126,19 @@ help: ## Show this help.
...
+
+kube_deploy:
+	@if [ -z $(KUBECONFIG) ]; then \
+        KUBECONFIG=$$HOME/.kube/config; \
+    fi;
+	@KUBECONFIG=$(KUBECONFIG) helm upgrade --install $(APPNAME) -n$(NAMESPACE) ./deploy/sigma --set image.repository=$(REPOSITORY),image.tag=$(TAG),mysql.auth.rootPassword=$(RANDOM_PASSWORD),mysql.auth.password=$(RANDOM_PASSWORD),redis.auth.password=$(RANDOM_PASSWORD),minio.secretKey=$(RANDOM_PASSWORD)
```
Comment (tosone) [inline Makefile:140]: Use backslashes to prevent each line from becoming too long.

File: Makefile [hunk 4]
```diff
@@ -121,3 +126,19 @@ help: ## Show this help.
...
+
+kube_undeploy:
+	@KUBECONFIG=$(KUBECONFIG)
+	@helm uninstall $(APPNAME) -n$(NAMESPACE)
```
Comment (tosone) [inline Makefile:144]: `helm --kubeconfig path/to/kubeconfig` is better here, same as the `kube_deploy`.

File: deploy/sigma/templates/_helpers.tpl [hunk 5]
```diff
@@ -42,10 +43,39 @@ app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
...
+
 {{/*
 Selector labels
 */}}
 {{- define "sigma.selectorLabels" -}}
 app.kubernetes.io/name: {{ include "sigma.name" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
 {{- end }}
+
+
```
Comment (tosone) [inline deploy/sigma/templates/_helpers.tpl:70]: remove this blank line. leave just one line.

File: deploy/sigma/templates/distribution/deployment.yaml [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+              port: {{ .Values.service.distribution.port }}
+          readinessProbe:
+            httpGet:
+              path: /
+              port: {{ .Values.service.distribution.port }}
+          volumeMounts:
+            - name: "config"
+              mountPath: "/etc/sigma/config.yaml"
+              subPath: "config.yaml"
+              readOnly: false
```
Comment (tosone) [inline deploy/sigma/templates/distribution/deployment.yaml:53]: readOnly should be true.

File: deploy/sigma/templates/worker/deployment.yaml [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+              port: {{ .Values.service.server.port }}
+          readinessProbe:
+            httpGet:
+              path: /
+              port: {{ .Values.service.server.port }}
+          volumeMounts:
+            - name: "config"
+              mountPath: "/etc/sigma/config.yaml"
+              subPath: "config.yaml"
+              readOnly: false
```
Comment (tosone) [inline deploy/sigma/templates/worker/deployment.yaml:53]: readonly should be true.

Comment (wang-chunxing) [reply deploy/sigma/templates/worker/deployment.yaml:53]: done

**Pull request merged**